Tests/scripts/run_parallel_pkg_dev_tasks.py

```diff
@@ -82,6 +90,7 @@ def main():
             res = future.result()
             handle_run_res(res, fail_pkgs, good_pkgs)
     if fail_pkgs:
+        create_result_files(fail_pkgs)
```

Comment:
can you explain the flow here?

Reply:
I want to get only the fail packages. In the circle it will look like this:
![image](https://user-images.githubusercontent.com/45915502/69607416-dc129700-102d-11ea-857c-c875f09bc047.png)

So I'm taking this packages and creating a file that later the slack_notifier will read and execute.

Reply:
ok got it
lets improve function name,and add docstring to it